test/client_test.py

```diff
@@ -97,12 +97,12 @@ def test_state_event():
 
     ev["type"] = "m.room.aliases"
     room._process_state_event(ev)
-    assert room.aliases is None
+    assert room.aliases == []
 
     aliases = ["#foo:matrix.org", "#bar:matrix.org"]
     ev["content"]["aliases"] = aliases
     room._process_state_event(ev)
-    assert room.aliases is aliases
+    assert room.aliases == aliases
```

Comment:
Since now we are processing list element by element, `is` will no longer work here.